go/ct/common/revisions.go

```diff
@@ -122,6 +122,23 @@ func GetForkBlock(revision Revision) (uint64, error) {
 	return 0, fmt.Errorf("unknown revision: %v", revision)
 }
 
+func GetForkTimestamp(revision Revision) (uint64, error) {
+	switch revision {
+	case R07_Istanbul:
+	case R09_Berlin:
+	case R10_London:
+	case R11_Paris:
+		return 0, fmt.Errorf("revision: %v is too low for timestamps", revision)
+	case R12_Shanghai:
+		return 4000, nil
```

Comment:
should we just for the sake of consistency use the same numbers as for `GetForkBlock` ?

Reply:
There are the same values as ForkBlock, I still do not know what are the implications here. 
It may be that we need to tune the timestamps later on. For now, I'll leave them as they are.